modules/tinymce/src/plugins/help/main/ts/data/PluginUrls.ts

```diff
@@ -73,6 +73,7 @@ const urls = Arr.map<PartialPluginUrl, PluginUrl>([
   { key: 'tinycomments', name: 'Tiny Comments', type: PluginType.Premium, slug: 'introduction-to-tiny-comments' },
   { key: 'tinydrive', name: 'Tiny Drive', type: PluginType.Premium, slug: 'tinydrive-introduction' },
   { key: 'tableofcontents', name: 'Table of Contents', type: PluginType.Premium },
+  { key: 'inlinecss', name: 'InlineCss', type: PluginType.Premium },
```

Comment:
Even though `tableofcontents` is still in the wrong place, we should at leas try to keep partial alphabetic ordering, so that the plugins are easier to find.